src/NUnitFramework/nunitlite.tests/MakeRunSettingsTests.cs

```diff
@@ -29,56 +29,31 @@ namespace NUnitLite.Tests
+        private static TestCaseData[] SettingsData =
         {
-            var options = new NUnitLiteOptions("test.dll", "--timeout=50");
-            var settings = TextRunner.MakeRunSettings(options);
-
-            Assert.That(settings.ContainsKey("DefaultTimeout"));
-            Assert.AreEqual(50, settings["DefaultTimeout"]);
-        }
+#if !NETCOREAPP1_1
+            new TestCaseData("--timeout=50", "DefaultTimeout", 50),
 #endif
-
-        [Test]
-        public void WhenWorkDirectoryIsSpecified_RunSettingsIncludeIt()
-        {
-            var options = new NUnitLiteOptions("test.dll", "--work=results");
-            var settings = TextRunner.MakeRunSettings(options);
-
-            Assert.That(settings.ContainsKey("WorkDirectory"));
-            Assert.AreEqual(Path.GetFullPath("results"), settings["WorkDirectory"]);
-        }
-
-        [Test]
-        public void WhenSeedIsSpecified_RunSettingsIncludeIt()
-        {
-            var options = new NUnitLiteOptions("test.dll", "--seed=1234");
-            var settings = TextRunner.MakeRunSettings(options);
-
-            Assert.That(settings.ContainsKey("RandomSeed"));
-            Assert.AreEqual(1234, settings["RandomSeed"]);
-        }
-
-        [Test]
-        public void WhenWorkersIsSpecified_RunSettingsIncludeIt()
+            new TestCaseData("--work=results", "WorkDirectory", Path.GetFullPath("results")),
+            new TestCaseData("--seed=1234", "RandomSeed", 1234),
+            new TestCaseData("--workers=8", "NumberOfTestWorkers", 8),
+            new TestCaseData("--prefilter=A.B.C", "LOAD", new string[] { "A.B.C" }),
+            new TestCaseData("--test=A.B.C", "LOAD", new string[] { "A.B.C" }),
+            new TestCaseData("--test=A.B.C(arg)", "LOAD", new string[] { "A.B.C" }),
+            new TestCaseData("--test=A.B<type>.C(arg)", "LOAD", new string[] { "A.B" })
+       };
+
+        [TestCaseSource(nameof(SettingsData))]
+        public void CheckSetting(string option, string key, object value)
         {
-            var options = new NUnitLiteOptions("test.dll", "--workers=8");
+            var options = new NUnitLiteOptions("test.dll", option);
             var settings = TextRunner.MakeRunSettings(options);
 
-            Assert.That(settings.ContainsKey("NumberOfTestWorkers"));
-            Assert.AreEqual(8, settings["NumberOfTestWorkers"]);
+            Assert.That(settings.ContainsKey(key));
+            Assert.AreEqual(value, settings[key]);
         }
 
-        [Test]
-        public void WhenLoadIsSpecified_RunSettingsIncludeIt()
-        {
-            var options = new NUnitLiteOptions("test.dll", "--prefilter=A.B.C");
-            var settings = TextRunner.MakeRunSettings(options);
-
-            Assert.That(settings.ContainsKey("LOAD"));
-            Assert.AreEqual(new string[] { "A.B.C" }, settings["LOAD"]);
-        }
+        //[Test]
+        //public void WhenTestFixtureIsSpecified_RunSettingsIncludeIt)
```

Comment:
Leftovers?

Reply:
Yup!

Reply:
@CharliePoole Oops, @oznetmaster [noticed](https://github.com/nunit/nunit/commit/d09def3521dca9be6fbd84fc483c555c4955afe3#r29649322) that the lines made it into master!

Reply:
Yes... I responded. I guess they can wait till somebody is in this file again.